documentation/modules/overview/con-configuration-points-broker.adoc

```diff
@@ -9,57 +9,58 @@ A kafka cluster comprises one or more brokers.
 For producers and consumers to be able to access topics within the brokers, Kafka configuration must define how data is stored in the cluster, and how the data is accessed.
 You can configure a Kafka cluster to run with multiple broker nodes across racks.
 
-Storage::
-Kafka and ZooKeeper store data on disks.
-+
-Strimzi requires block storage provisioned through `StorageClass`.
-The file system format for storage must be _XFS_ or _EXT4_.
-Three types of data storage are supported:
-+
---
+== Storage
+Strimzi supports the following storage configuration options for managing Kafka and ZooKeeper data:
+
 Ephemeral (Recommended for development only):: Ephemeral storage stores data for the lifetime of an instance. Data is lost when the instance is restarted.
 Persistent:: Persistent storage relates to long-term data storage independent of the lifecycle of the instance.
 JBOD (Just a Bunch of Disks, suitable for Kafka only):: JBOD allows you to use multiple disks to store commit logs in each broker.
---
-+
+Tiered storage (Early access):: Tiered storage is an early access feature in Kafka. Tiered storage provides more flexibility for data management by leveraging the parallel use of storage types with different characteristics.
```

Comment:
I don't think this is correct. It presents the tiered storage as alternative. But it is not. You have to always use one of the 3 basic options. And on top of that, you can use the tiered storage. But you cannot decide that you are going to use only tiered storage.

Reply:
Right. I've removed from the list and described it more as an add-on.